x-pack/plugins/maps/public/components/metrics_editor/validated_number_input.tsx

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Can you move this to top level components folder so it can be used more broadly?